### ISSUE TITLE
Number Ranges can't be 0

### ISSUE THREAD

**Issue by Daenara** [rhasspy/rhasspy#252]
I tried implementing my shutter control and was testing different sentences when I thought I would ask to set the shutter to 0 instead of asking to close the shutter, just to test that sentence part. The result was an empty value for the slot.

```
[ShutterControl]
(mach | mache | stelle) den rollladen ((hoch | auf):open | (zu | runter):close) {position}
stelle den rollladen auf [position] (jalousie:10 | (0..100)) {position} [ein]
```

`stelle den rollladen auf 0` results in this:
```
{
    "entities": [
        {
            "end": 24,
            "entity": "position",
            "raw_end": 29,
            "raw_start": 25,
            "raw_value": "null",
            "start": 25,
            "value": "",
            "value_details": {
                "kind": "Unknown",
                "value": ""
            }
        }
    ],
    "intent": {
        "confidence": 1,
        "name": "ShutterControl"
    },
    "raw_text": "stelle den rollladen auf 0",
    "raw_tokens": [
        "stelle",
        "den",
        "rollladen",
        "auf",
        "0"
    ],
    "recognize_seconds": 0.08451609400071902,
    "slots": {
        "position": ""
    },
    "speech_confidence": 1,
    "text": "stelle den rollladen auf",
    "tokens": [
        "stelle",
        "den",
        "rollladen",
        "auf"
    ],
    "wakeword_id": null
}
```

As opposed to `stelle den rollladen auf 1` which results in:
```
{
    "entities": [
        {
            "end": 26,
            "entity": "position",
            "raw_end": 29,
            "raw_start": 25,
            "raw_value": "eins",
            "start": 25,
            "value": 1,
            "value_details": {
                "kind": "Number",
                "value": 1
            }
        }
    ],
    "intent": {
        "confidence": 1,
        "name": "ShutterControl"
    },
    "raw_text": "stelle den rollladen auf 1",
    "raw_tokens": [
        "stelle",
        "den",
        "rollladen",
        "auf",
        "1"
    ],
    "recognize_seconds": 0.17387051100376993,
    "slots": {
        "position": 1
    },
    "speech_confidence": 1,
    "text": "stelle den rollladen auf 1",
    "tokens": [
        "stelle",
        "den",
        "rollladen",
        "auf",
        "1"
    ],
    "wakeword_id": null
}
```

For a test I set the range to (-1..100) hoping it would include the 0, but it still results in an empty slot. My rhasspy version is 2.5.11 and I did not notice it before with my older sentence that uses a number range with 0 because the backend that reacts to that is not finished, so it could be an older bug.

**Comment by Romkabouter** [comment]
Hi there, 

Are you still having this issue? Because when I try it with english language it seems to work ok:

![image](https://user-images.githubusercontent.com/4774478/143128193-5530eee7-59cd-450f-80be-5638c6b57f67.png)

**Comment by Daenara** [comment]
I just tested it and it works. Speak of bugs spontaneously fixing themselfs.

**Comment by synesthesiam** [comment]
Not spontaneous, I just forgot to post about it! :no_mouth:

**Comment by Daenara** [comment]
> Not spontaneous, I just forgot to post about it! 😶

Then thank you for that. I just assumed it fixed itself because I reported on 2.5.11 and I still am on 2.5.11 and I can't actually remember running an update, but apparently I did.
